base/uk.ac.stfc.isis.ibex.ui.configserver/src/uk/ac/stfc/isis/ibex/ui/configserver/commands/NewBlockHandler.java

```diff
@@ -47,13 +47,16 @@
 import uk.ac.stfc.isis.ibex.ui.configserver.ConfigurationViewModels;
 import uk.ac.stfc.isis.ibex.ui.configserver.editing.blocks.EditBlockDialog;
 
+import org.apache.logging.log4j.Logger;
```

Comment:
The changes in this file are junk, please make sure you only commit changes that are part of resolving the ticket